### PR TITLE
docs(redact): adding doc blocks

### PR DESCRIPTION
**Pull request by manchuck**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/redact/package.json [hunk 1]
```diff
@@ -11,7 +12,16 @@
     "url": "git+https://github.com/Vonage/vonage-node-sdk.git"
```
Comment (pardel) [inline packages/redact/package.json:12]: Is this the right URL?

File: packages/redact/package.json [hunk 2]
```diff
@@ -11,7 +12,16 @@
...
   },
   "license": "Apache-2.0",
-  "author": "Chuck MANCHUCK Reeves <chuck@manchuck.com>",
+  "contributors": [
+    {
+      "name": "Chuck MANCHUCK Reeves",
```
Comment (pardel) [inline packages/redact/package.json:17]: please use vonage email addresses

**Review by pardel — CHANGES_REQUESTED** [review]

File: packages/redact/package.json (hunk 1 above)
Comment (manchuck) [reply packages/redact/package.json:12]: Yup for all monorepos it will be this

File: packages/redact/package.json (hunk 2 above)
Comment (manchuck) [reply packages/redact/package.json:17]: will update when all doc block PRs are merged in

**Pull request merged**